app/assets/javascripts/services/miq_service.js

```diff
@@ -53,6 +54,14 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', funct
     $('#flash_msg_div').text("");
   };
 
+  this.miqFlashLater = function(msgObj) {
```

Comment:
A couple of things are happening here.

`miqService` is our generic helper service that mostly contains common code that the angular controllers consume. Flash messages happens to be one part of the common code. 

To be able to use `miqFlashLater` in the physical server toolbar component, I had to declare it as a function in `miqService`.
By doing this, 
- I have added one more Flash function to the existing family of Flash functions in `miqService`
- I have added `miqFlashLater` in the global comment here https://github.com/AparnaKarve/manageiq-ui-classic/blob/0d7737978e05fc9e1a45f8b56371c6889151fe7b/app/assets/javascripts/services/miq_service.js#L2, which actually addresses a CC error
Note that, CC discourages the use of global methods in angular code and this is the only way to keep the CC error at bay.